scst-store/use-external-database.hbs.md

```diff
@@ -1,43 +1,56 @@
 # Use external PostgreSQL database for Supply Chain Security Tools - Store
 
-This topic describes how you can configure and use your external PostgreSQL database for Supply Chain Security Tools (SCST) - Store.
+For production deployments, it is recommended to use an external Postgres database rather the one packaged with the Tanzu Application Platform.  This will allow you to manage it via best practices and process that your organization have established.  This topic describes how you can configure your TAP deployment to use an external database.
 
 ## <a id='prereqExtrenalDB'></a>Prerequisites
 
-- Set up your external PostgreSQL database. After the database instance starts, retrieve the following information:
+- Gather the connection information for the external PostgreSQL database: 
 
-   1. Database Instance Endpoint
-   2. Main User name
-   3. Main Password
-   4. Database Name
+   1. Database Instance Connect Endpoint
+   2. (Optional if TLS is enabled and certificate is self signed) CA Certificate that signed the certificate
```

Comment:
`Optional if TLS is enabled and certificate is self signed` sounds like it is optional when tls is enabled and cert is self signed. Maybe remove the `Optional` and just say `If TLS is enabled and certificate is self signed: CA Certificate that signed the certificate`

Reply:
Thanks for the feedback.  That sentence definitely was weird.  I completely re-route it for more clarify